src/components/Page/Toolbar/Toolbar.jsx

```diff
@@ -143,7 +169,7 @@ export class Toolbar extends Component {
             <PaintSettings
               initialColor={`rgb(${r * 255}, ${g * 255}, ${b * 255})`}
               onPickColor={color => {
-                boardState.drawBoard.changeBrushColor([color.rgb.r / 255, color.rgb.g / 255, color.rgb.b / 255]);
+                boardState.drawBoard.changeToolSetting(`${boardState.tool}Color`,[color.rgb.r / 255, color.rgb.g / 255, color.rgb.b / 255]);
```

Comment:
Using `${boardState.tool}Color` can break things because the tool can be eraser or rainbow_brush too. It would be better if we check for the line or brush tools specifically.

Reply:
Ok sure👍🏻